### PR TITLE
feat(CSSClass): support binding to classList

### LINKED ISSUE #876: Unable to set the property 'classList' on an element
This needs to be implemented.
```
@Decorator({
  selector: '[class]',
  bind: {
    'classChanges': 'class | keyValueDiff'
  }
})
class ClassDecorator {
  element:NgElement;
  constructor(element:NgElement) {
    this.element = element;
  }

  classChanges(changes:KeyValueChanges) {
    changes.forEachChangedItem(...)
  }
}

@Decorator({
  selector: '[class-list]',
  bind: {
    'classChanges': 'classList | iterableDiff'
  }
})
class ClassDecorator {
  element:NgElement;
  constructor(element:NgElement) {
    this.element = element;
  }

  classChanges(changes:ArrayChanges) {
    changes.forEachChangedItem(...)
  }
}
```

### Original Text

I tried both

```html
<div [class-list]="'one two'"></div>
```
and

```html
<div [classList]="'one two'"></div>
```

at element_binder_builder.dart:158, the first example is checking the property 'class-list' on the JS element.
The second example is checking the property 'classlist'.

I would expect it to set the property 'classList'.

This is with the Dart release 2.0.0-alpha-12

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**
Closes #876

### CODE REVIEW

**Comment by pkozlowski-opensource** [comment]
Adding to M7 as the issue it is fixing is in M7

File: modules/angular2/src/directives/class.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    'iterableChanges': 'class | keyValDiff'
+  }
+})
+export class CSSClass {
+  _domEl;
+  constructor(ngEl: NgElement) {
+    this._domEl = ngEl.domElement;
+  }
+
+  _toggleClass(record) {
```
Comment (vicb) [inline modules/angular2/src/directives/class.js]: `_toggleClass(className, enabled)` ?

**Comment by vicb** [comment]
lgtm with one minor comment

**Comment by pkozlowski-opensource** [comment]
Merged as aca4604879c239010c071fc2f63ca084e493ca20, thnx for the review @vicb !

**Pull request closed without merging**